rest-client/src/main/java/org/sagebionetworks/bridge/sdk/rest/Utilities.java

```diff
@@ -20,16 +23,22 @@ static ObjectMapper getObjectMapper() {
         mapper.configure(SerializationFeature.WRITE_DATES_AS_TIMESTAMPS, false);
```

Comment:
I didn't notice this earlier, but we've introduced Jackson into rest-client. Do we want to use/support both Gson and Jackson? Or is this temporary?